src/data/pokemon/evolution.h

```diff
@@ -466,15 +466,15 @@ const struct Evolution gEvolutionTable[NUM_SPECIES][EVOS_PER_MON] =
     [SPECIES_YAMPER]                = {{EVO_LEVEL, 25, SPECIES_BOLTUND}},
     [SPECIES_ROLYCOLY]              = {{EVO_LEVEL, 18, SPECIES_CARKOL}},
     [SPECIES_CARKOL]                = {{EVO_LEVEL, 34, SPECIES_COALOSSAL}},
-    [SPECIES_APPLIN]                = {{EVO_ITEM, ITEM_NONE,/*ITEM_TART_APPLE*/ SPECIES_FLAPPLE},
-                                       {EVO_ITEM, ITEM_NONE,/*ITEM_SWEET_APPLE*/ SPECIES_APPLETUN}},
+    [SPECIES_APPLIN]                = {{EVO_ITEM, ITEM_NONE, SPECIES_FLAPPLE},
+                                       {EVO_ITEM, ITEM_NONE, SPECIES_APPLETUN}},
     [SPECIES_SILICOBRA]             = {{EVO_LEVEL, 36, SPECIES_SANDACONDA}},
     [SPECIES_ARROKUDA]              = {{EVO_LEVEL, 26, SPECIES_BARRASKEWDA}},
     [SPECIES_TOXEL]                 = {{EVO_LEVEL, 30, SPECIES_TOXTRICITY},
                                        {EVO_LEVEL, 30, SPECIES_TOXTRICITY_LOW_KEY}},
     [SPECIES_SIZZLIPEDE]            = {{EVO_LEVEL, 28, SPECIES_CENTISKORCH}},
     [SPECIES_CLOBBOPUS]             = {{EVO_MOVE, MOVE_TAUNT, SPECIES_GRAPPLOCT}},
-    [SPECIES_SINISTEA]              = {{EVO_ITEM, ITEM_NONE,/*ITEM_CRACKED_POT*/ SPECIES_POLTEAGEIST}},
+    [SPECIES_SINISTEA]              = {{EVO_ITEM, ITEM_CRACKED_POT, SPECIES_POLTEAGEIST}},
```

Comment:
```suggestion
    [SPECIES_SINISTEA]              = {{EVO_ITEM, ITEM_NONE, SPECIES_POLTEAGEIST}},
```
Breaks if merged with Item Expansion since it's not defined in Item Expansion yet.